spring-cloud-skipper-server-core/pom.xml

```diff
@@ -185,6 +186,25 @@
             <groupId>org.springframework.security.oauth</groupId>
             <artifactId>spring-security-oauth2</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Well this smells a little as we used to have cf/k8s deps in a server core and moved those into separate packages. Is it possible to keep everything in `spring-cloud-skipper-platform-cloudfoundry` so that core doesn't need to know anything about cf code?